layouts/partials/admonition.html

```diff
@@ -0,0 +1,23 @@
+{{ $content := .content | markdownify }}
```

Comment:
We had out admonitions? :)

Reply:
Sorry, I don't quite understand the question

Reply:
Oops, meant *hand, as in "hand out". I just found it funny to have a partial named admonition -- I haven't come across the word in quite a while. 

So there was no serious question or comment underlying my first remark.

Reply:
Ooooh, okay, gotcha haha. Yeah, "admonition" is definitely a term that only tech writers use. I got the terminology from [rST](https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions).